.github/workflows/build_release.yml

```diff
@@ -0,0 +1,115 @@
+name: Build and release
+
+on:
+  pull_request:
+    types: [closed]
+    branches: 
+      - "develop" # we can delete this later
+      - "main"
+jobs:
+  Build-release-package:
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.merged == true || github.event.review.state == 'approved'
+    steps:
+      - uses: actions/checkout@v2
+
+      ### RELEASE VERSION SECTION ###
+      - name: Get current version (MAJOR.MINOR) from DESCRIPTION file
+        id: get_version
+        run: |
+          # Extract version from DESCRIPTION file
+          VERSION=$(grep '^Version:' DESCRIPTION | awk '{print $2}')
+          MAJOR=$(echo $VERSION | cut -d '.' -f 1)
+          MINOR=$(echo $VERSION | cut -d '.' -f 2)
+          
+          # Output MAJOR and MINOR to GitHub environment
+          echo "MAJOR=$MAJOR" >> $GITHUB_ENV
+          echo "MINOR=$MINOR" >> $GITHUB_ENV    
+          echo "The version from DESCRIPTION: $MAJOR.$MINOR"
+
+      - name: Get latest PATCH version from GitHub releases
+        id: get_latest_patch
+        run: |
+          # Get the list of existing releases
+          releases=$(curl -s "https://api.github.com/repos/${{ github.repository }}/releases")
+      
+          # Find the latest release with the same MAJOR.MINOR version
+          latest_patch=$(echo "$releases" | jq -r --arg MAJOR "$MAJOR" --arg MINOR "$MINOR" '
```

Comment:
What happens when Major.Minor is updated?

Reply:
the developer might need to update the version in DESCRIPTION file